<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0,0">
   <item>
    <widget class="QLabel" name="labelCopyingFiles">
     <property name="text">
      <string>Copying n items (size)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlProgress">
     <item>
      <widget class="QLabel" name="labelProgress">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>n% complete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelETA">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ETA:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frameSpeedGraph">
     <property name="frameShape">
      <enum>QFrame::Shape::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="SpeedGraph" name="speedGraphWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelFrom">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTo">
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
     <item>
      <widget class="QLabel" name="labelItems">
       <property name="text">
        <string>Items remaining: n (size)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPause">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Pause</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Cancel</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ListRemove"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpeedGraph</class>
   <extends>QWidget</extends>
   <header>MainWindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
